### PR TITLE
Add ViralConsensus 0.0.3

### PR DESCRIPTION
**Pull request by daniel-ji**

### CODE REVIEW

File: tools/htslib/src
Comment (robertaboukhalil) [inline tools/htslib/src:1]: Lets keep the default `htslib` at tag 1.17. I _think_ `git checkout 1.17` then committing that change should do the trick.

File: tools/ViralConsensus/compile.sh [hunk 1]
```diff
@@ -1,6 +1,8 @@
+sed -i 's/unsigned char \*tmp = realloc(b->data, len);/unsigned char \*tmp = static_cast<unsigned char \*>(realloc(b->data, len));/g' htslib/cram/cram_io.h
```
Comment (robertaboukhalil) [inline tools/ViralConsensus/compile.sh:1]: Thanks! Can you add a comment on top briefly mentioning why that change is needed?

**Review by robertaboukhalil — CHANGES_REQUESTED** [review]
Thanks so much Daniel! Just a few minor changes

File: tools/ViralConsensus/compile.sh (hunk 1 above)
Comment (daniel-ji) [reply tools/ViralConsensus/compile.sh:2]: Done!

**Review by robertaboukhalil — APPROVED** [review]
Looks great, thanks!

**Pull request merged**